manifest/testdata/attribute_row.md

```
@@ -1,2 +1,3 @@
-[96m[96mName   [90m[90m | [0m[96m[0m[96mDOM Property[90m[90m | [0m[96m[0m[96mReflects[90m[90m | [0m[96m[0m[96mSummary[90m[90m | [0m[96m[0m[96mDefault[90m[90m | [0m[96m[0m[96mType                   [90m[90m | [0m[96m[0m[96mInherited From[0m
-[96m[0m[0mvariant[90m[90m | [0m[0m            [90m[90m | [0m[0m        [90m[90m | [0m[0m       [90m[90m | [0m[0m       [90m[90m | [0m[0m"primary" | "secondary"[90m[90m | [0m[0m              
+Name    | DOM Property | Reflects | Summary | Default | Type                    | Inherited From
+------- | ------------ | -------- | ------- | ------- | ----------------------- | --------------
+variant |              |          |         |         | "primary" | "secondary" |               
```

markdownlint-cli2 (0.22.1)

[warning] Table pipe style
Expected: no_leading_or_trailing; Actual: trailing_only; Unexpected trailing pipe

(MD055, table-pipe-style)